client.py

```diff
@@ -0,0 +1,152 @@
+""" SHEST GUI CLIENT """
+
+from CONSTANTS import *
+from cryptography.fernet import Fernet
+import requests
+from threading import Thread
+import time
+import tkinter
+
+
+def load_key():
+    """Load the previously generated key"""
+
+    with open('secret.key', 'r') as file:
+        return file.read()
+
+
+def encrypt_message(message: str):
+    """
+    Encrypts a message"
+    :param message: A regular message.
+    :return: The encrypted message.
+    """
+
+    key = load_key()
+    f = Fernet(key)
+    message = message.encode()
+    encrypted_message = f.encrypt(message)
+    return encrypted_message
+
+
+def decrypt_message(encrypted_message):
+    """
+    Decrypts an encrypted message
```

Comment:
Note the grammar of function title docs. The tense should be "what the function should do when called".